src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -54,6 +54,11 @@ function FrameLocation({ frame, displayFullUrl = false }: FrameLocationProps) {
     <span className="location">
       <span className="filename">{filename}</span>:
       <span className="line">{location.line}</span>
+      {location.column && (
+        <span>
```

Comment:
Can we avoid the extra div here? A React.Fragment might work.

Reply:
Thanks, I was getting a warning from React to counter - I added the extra span.